yapapi/props/base.py

```diff
@@ -316,6 +320,8 @@ def join_str_constraints(constraints: List[str], operator: ConstraintGroupOperat
         (bar<=128))
     ```
     """
+    constraints = list(filter(bool, constraints))
```

Comment:
```suggestion
    constraints = [c for c in constraints if c]
```
I think this is more pythonic? At least I've never seen `list(filter(bool` before.